pennylane/tape/tape.py

```diff
@@ -189,18 +197,31 @@ def stop_at(obj):  # pylint: disable=unused-argument
                 except DecompositionUndefinedError:
                     # Object does not define an expansion; treat this as
                     # a stopping condition.
-                    getattr(new_tape, queue).append(obj)
+                    new_queue.append(obj)
                     continue
 
             # recursively expand out the newly created tape
-            expanded_tape = expand_tape(obj, stop_at=stop_at, depth=depth - 1)
+            exp_prep, exp_ops, exp_measurements = expand_tape(
+                obj, stop_at=stop_at, depth=depth - 1, iterative_call=True
+            )
```

Comment:
Here we use `iterative_call=True`, expecting three returned objects instead of a tape.